src/github.com/eriklupander/gotling/httpreq.go

```diff
@@ -1,26 +1,25 @@
 package main
```

Comment:
no function change in this file, 
just want to confirm with you about the format style, for example:
> Indentation
We use tabs for indentation and gofmt emits them by default. Use spaces only if you must.

you can see the the original code mixed up tabs and spaces, for now, i use spaces over tabs which make few changes.

actually i'd prefer with the gofmt way...